### PR TITLE
Change EventArgs to Argument

### PR DESCRIPTION
**Pull request by hulumane**
One more final API change

### CODE REVIEW

**Review by pmpurifoy — APPROVED** [review]

**Review by robertzhou-wpn — APPROVED** [review]

File: dev/AppNotifications/AppNotificationActivatedEventArgs.h [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         AppNotificationActivatedEventArgs() = default;
 
         AppNotificationActivatedEventArgs(winrt::hstring const& arguments, winrt::Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> const& userInput) : m_arguments(arguments), m_userInput(userInput) {};
-        winrt::hstring ActivationArgs() { return m_arguments; };
+        winrt::hstring Argument() { return m_arguments; };
```
Comment (loneursid) [inline dev/AppNotifications/AppNotificationActivatedEventArgs.h:11]: Why Argument() singular and not Arguments() plural?

Comment (hulumane) [reply dev/AppNotifications/AppNotificationActivatedEventArgs.h:11]: Because Arguments imply its a vector. Argument implies a singleton.

Comment (hulumane) [reply dev/AppNotifications/AppNotificationActivatedEventArgs.h:11]: API Review feedback ;)

Comment (hulumane) [reply dev/AppNotifications/AppNotificationActivatedEventArgs.h:11]: Also aligns with what we have in Windows SDK today:
    [marshaling_behavior(standard)]
    [contract(Windows.Foundation.UniversalApiContract, 1)]
    runtimeclass ToastNotificationActionTriggerDetail
    {
        [contract(Windows.Foundation.UniversalApiContract, 1)]
        [interface_name("Windows.UI.Notifications.IToastNotificationActionTriggerDetail", 9445135a-38f3-42f6-96aa-7955b0f03da2)]
        {
            String Argument{ get; };
            Windows.Foundation.Collections.ValueSet UserInput{ get; };
        }
    }

**Pull request merged**